modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -126,6 +134,12 @@ ifeval::[{suma-content} == true]
 Add 100{nbsp}MB per {suse} product, 1{nbsp}GB per {redhat} or other product.
 Double the space if the server is an ISS Master.
 
+|
+| [path]``/var/lib/containers/storage/volumes/srv-www``
+| Minimum 100 GB
+
+* Storage requirments should be calculated for the number of ISO distribution images, containers, and bootstrap repositories you will use.
+
```

Comment:
Same here, see bug report https://bugzilla.suse.com/show_bug.cgi?id=1230503

Just mention [path]``/var/lib/containers/storage/volumes/` to have at minimum 100GB+ and nothing else.

If we want to specify all volumes with their expected sized, we should do that in some advanced tab for advanced users.

Reply:
ok, but for the moment, it is disabled anyway (power).  We should update it, if we enable power again.